web-app/src/app/store/saga/auth-saga.ts

```diff
@@ -191,11 +191,13 @@ function* resetPasswordSaga({
 
 function* anonymousLoginSaga(): Generator {
   try {
+    const auth = getAuth();
+    const firebaseUser = app.auth().currentUser;
     // Check if the user is already authenticated
     const isAuthenticated: boolean = (yield select(
       selectIsAuthenticated,
     )) as boolean;
-    if (isAuthenticated) {
+    if (isAuthenticated && firebaseUser !== null) {
```

Comment:
@davidgamez This was added because there are certain cases during the tests where there was an authenticated user but no firebase user. This lead to the App being stuck in a state of not ready which leads to having just a white screen

I suspect this is the fix, and that it will fix the ResetPassword cypress test as well

Reply:
Great finding!